test/e2e/nix/build_suite_test.go

```diff
@@ -0,0 +1,104 @@
+package nix_test
+
+import (
+	"github.com/benchkram/bob/bob/bobfile"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+
+	"github.com/benchkram/bob/bob"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var (
+	dir     string
+	version string
+
+	stdout *os.File
+	pr     *os.File
+	pw     *os.File
+
+	b *bob.B
+
+	initialPath string
+)
+
+var _ = BeforeSuite(func() {
+	version = bob.Version
+	bob.Version = "1.0.0"
+
+	initialPath = os.Getenv("PATH")
+
+	//Initialize mock bob files from local directory
+	mocks := []string{
```

Comment:
Not really mocks.. rather test environment.

Reply:
I'll rename them to `bobFiles`.